### ISSUE TITLE
bignumber.min.js is missing in npm repo

### ISSUE THREAD

**Issue by avsrinivasa** [MikeMcl/bignumber.js#311]
The latest version of bignumber.js  (9.0.2) is missing with bignumber.min.js. As many of the other libraries are refering the bignumber.min.js file. This is causing the failure of Angular project production builds. Can you please add the bignumber.min.js file in npm?
Regards,
Srinivas

**Comment by MikeMcl** [comment]
Yes, apologies, I should have bumped the major version number, so the latest release would have been v10.0.0 rather than v9.0.2. 

A minified version is available from, for example:

 [https://cdn.jsdelivr.net/npm/bignumber.js@9.0.2/bignumber.min.js](https://cdn.jsdelivr.net/npm/bignumber.js@9.0.2/bignumber.min.js)
 
I no longer want to have the responsibility and maintenance burden of providing a minified version, so I won't be publishing one myself.
